ddmrp/README.rst

```diff
@@ -0,0 +1,242 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
```

Comment:
maybe switch to new readme structure?

Reply:
We are still not ready to move to OCA unfortunately. We are still working on some important improvements you can follow here: #69. But for sure, when the code is ready I will update the readme!